install_config/topics/ocp_osp_deploy_instance.adoc

```diff
@@ -0,0 +1,275 @@
+| id                  | 2bc06e39-1efb-453e-8642-39f910ac8fd1 |
+| port_id             | None                                 |
+| project_id          | ca304dfee9a04597b16d253efd0e2332     |
+| project_id          | ca304dfee9a04597b16d253efd0e2332     |
+| revision_number     | 1                                    |
+| router_id           | None                                 |
+| status              | DOWN                                 |
+| updated_at          | 2017-08-24T22:44:03Z                 |
++---------------------+--------------------------------------+
+----
+
+Within the above output, the `floating_ip_address` field shows that the floating
+IP `10.20.120.150` is created. In order to assign this IP to the deployment instance,
+ run the following command:
+
+[subs=+quotes]
+----
+$ source /path/to/examplerc
+$ openstack server add floating ip *<deployment-instance-name> <ip>*
+----
+
+For example, if instance `deployment.example.com` is to be assigned IP
+`10.20.120.150` the command would be:
+
+----
+$ source /path/to/examplerc
+$ openstack server add floating ip deployment.example.com 10.20.120.150
+----
+
+===== Adding the RC File to the Deployment Host
+
+Once the deployment host exists, copy the RC file created earlier to the
+deployment host via `scp` as follows
+
+----
+scp <rc-file-deployment-host> cloud-user@<ip>:/home/cloud-user/
+----
+
+==== Deployment Host Configuration for {product-title}
+
+The following subsections describe all the steps needed to properly configure
+the deployment instance.
+
+[[ssh_forwarding]]
+====== Configure ~/.ssh/config to use Deployment Host as a Jumphost
+
+To easily connect to the {product-title} environment, follow the steps below.
+
+On the OpenStack director node or local workstation with the private key,
+<keypair-name>.pem:
+
+----
+$ exec ssh-agent bash
+
+$ ssh-add /path/to/<keypair-name>.pem 
+Identity added: /path/to/<keypair-name>.pem (/path/to/<keypair-name>.pem)
+----
+
+Add to the `~/.ssh/config` file:
+
+----
+Host deployment
+    HostName        <deployment_fqdn_hostname OR IP address>
+    User            cloud-user
+    IdentityFile    /path/to/<keypair-name>.pem
+    ForwardAgent     yes
+----
+
+`ssh` into the deployment host with the `-A` option that enables forwarding of
+the authentication agent connection.
+
+Ensure the permissions are read write only for the owner of the _~/.ssh/config_
+file:
+
+----
+$ chmod 600 ~/.ssh/config
+----
+
+----
+$ ssh -A cloud-user@deployment
+----
+
+Once logged into the deployment host, verify the ssh agent forwarding is working
+via checking for the `SSH_AUTH_SOCK`
+
+----
+$ echo "$SSH_AUTH_SOCK"
+/tmp/ssh-NDFDQD02qB/agent.1387
+----
+
+====== Subscription Manager and Enabling {product-title} Repositories
+
+Within the deployment instance, register it with the Red Hat Subscription 
+Manager. This is accomplished via using credentials or an activation key.
+
+Via credentials:
```

Comment:
This can be accomplished by using credentials:

----
$ sudo subscription-manager register --username <user> --password '<password>'
----

Alternatively, you can use an activation key: